python/sglang/srt/server_args.py

```diff
@@ -1923,9 +1923,8 @@ def add_cli_args(parser: argparse.ArgumentParser):
         parser.add_argument(
             "--hicache-storage-backend",
             type=str,
-            choices=["file", "mooncake", "hf3fs", "nixl"],
```

Comment:
I think it's probably fine to still keep the choices

Reply:
If backend options are hardcoded here, it seems impossible to dynamically select the names of other backends?